code/midtown/pcwindis/dxinit.cpp

```diff
@@ -220,7 +220,7 @@ Ptr<agiSurfaceDesc> dxiScreenShot()
     Ptr<agiSurfaceDesc> surface =
         as_ptr agiSurfaceDesc::Init(width, height, agiSurfaceDesc::FromFormat(PixelFormat_B8G8R8));
 
-    void (*translate)(u8 * output, u16 * input, u32 width) = nullptr;
+    void (*translate)(u8* output, u16* input, u32 width) = nullptr;
```

Comment:
Unfortunately different versions of clang-format can produce slightly different results. Since this isn't part of the code you changed can you revert this?